index.d.ts

```diff
@@ -205,20 +208,20 @@ declare module 'gdax' {
         placeOrder(params: OrderParams, callback: callback<OrderResult>): void;
         placeOrder(params: OrderParams): Promise<OrderResult>;
 
-        cancelOrder(orderID: any, callback: callback<string>): void;
-        cancelOrder(orderID: any): Promise<string>;
+        cancelOrder(orderID: string, callback: callback<string | ErrorMessage>): void;
+        cancelOrder(orderID: string): Promise<string | ErrorMessage>;
```

Comment:
The promises don't actually resolve to errors

Reply:
It looks like my previous commit fixed the promise part but missed the orderID part. A rebase against master should point these out and fix them.